default.nix

```diff
@@ -49,4 +52,19 @@ pkgs.stdenv.mkDerivation {
     "fortify"
     "fortify3"
   ];
+
+  # Running Hyrise executables requires the LD_LIBRARY_PATH variable set. This does not
+  # happen by other hooks, therefore it happens manually here. Each of the derivations
+  # listed here provides a dynamically linked library.
+  # View the results by running `nix-shell --pure --run "echo $LD_LIBRARY_PATH"`.
+  shellHook = ''
+	export LD_LIBRARY_PATH="${pkgs.lib.makeLibraryPath [
+		pkgs.lld
+		pkgs.sqlite
+		pkgs.tbb_2021_11
+		pkgs.boost
+		pkgs.libgccjit
```

Comment:
What is that?

Reply:
That is required for running the executables after building. It has something to do with dynamic linking of libraries, but I am too little of an expert for that 😅 For some reason, it is only required in Linux.

Reply:
Can you show me the failed commit when it is missing?

Reply:
It is `58d6b17`.